oxide-auth/src/primitives/scope.rs

```diff
@@ -72,20 +72,20 @@ impl Scope {
     fn invalid_scope_char(ch: char) -> bool {
         match ch {
             '\x21' => false,
-            ch if ch >= '\x23' && ch <= '\x5b' => false,
-            ch if ch >= '\x5d' && ch <= '\x7e' => false,
-            ' ' => false, // Space seperator is a valid char
+            ch if ('\x23'..='\x5b').contains(&ch) => false,
+            ch if ('\x5d'..='\x7e').contains(&ch) => false,
+            ' ' => false, // Space separator is a valid char
             _ => true,
         }
     }
 
     /// Determines if this scope has enough privileges to access some resource requiring the scope
     /// on the right side. This operation is equivalent to comparison via `>=`.
-    pub fn priviledged_to(&self, rhs: &Scope) -> bool {
+    pub fn privileged_to(&self, rhs: &Scope) -> bool {
```

Comment:
Breaking, but can also be solved with a hidden/deprecated method. Weirdly, I have a faint memory of removing this at some point but it may have been somewhere else.